README.md

```diff
@@ -1,11 +1,173 @@
 # Jormungandr
 Main Software Repository for Subbots
 
-To Run UWSim: </br>
-  1: Run install_dependencies.sh script in main repository. </br>
-  2: Add source ~/Jormungandr/devel/setup.sh to .bashrc, or source this file every time. </br>
-  3: Run ~/Jormungandr/src/simulator/run_simulator.sh "scene" (Where "scene" is one of the xml scenes in Jorgumandr/src/simulator/scenes ie. Qualification.xml) </br>
+![alt tag](https://travis-ci.org/ubc-subbots/Jormungandr.svg?branch=master)
 
+## Table of Contents
+- [Setup](#setup)
+  - [Rosinstall](#rosinstall)
+- [Conventions](#conventions)
+  - [Github](#github-conventions)
+  - [Coding Conventions](#coding-conventions)
+  - [Coordinate Systems](#coordinate-systems)
+- [Creating a New Node](#creating-a-new-node)
+- [Creating a New Simple Node](#creating-a-new-simple-node)
+- [Launch Files](#launch-files)
+- [Testing](#testing)
+  - [GTest](#gtest)
+  - [Rostest](#rostest)
+- [Running Packages](#running-packages)
+  - [Running UWSim](#to-run-uwsim)
+  - [Running Phidget IMU](#running-phidget)
+  - [Running Camera Driver](#cammera-driver)
 
-![alt tag](https://travis-ci.org/ubc-subbots/Jormungandr.svg?branch=master)
 
+## Setup 
+
+### Rosinstall
+
+Rostinstall is addressed and run in `install_dependencies.sh`, should there be any need to run manual, rosinstall should be run after all steps from installation and setup has been completed. 
+
+## Conventions
+
+### Github Conventions
+- We follow the Forking Workflow: here is what it is [here](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) and how to use it [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- Only commit files that are essential for the system to run; do not put any photos or videos in here
+- All files **must** be formatted properly. Formatting will be enforced with the `clang-format` tool. 
+    - To check and fix formatting, from the `Snowflake` folder run `./clang_format/fix_formatting.sh BRANCH_NAME`, where `BRANCH_NAME` is the name of the branch you intend to merge your code into (ex. `iarrc` or `core`). This script will fix any improperly formatted code, but will refuse to change any files with uncommited changes (to prevent you losing work)
```

Comment:
we could perhaps change "`iarrc` and `core`" to `master` (just reduce potential confusion as to why neither of the branches given as examples exist in the repo)

Reply:
`Snowflake` should be changed to `Jormungandr` (I'm just nothing this one instance, but all instances should be changed)